testgen/generators/rdt_fmt_gen.js

```diff
@@ -95,12 +156,29 @@ function generateAll() {
           all_options['numberingSystem'] = number_system;
         }
 
-        let formatter;
+        let all_options_numeric_always = {...all_options};
+        all_options_numeric_always['numeric'] = 'always';
+
+        let all_options_numeric_auto = {...all_options};
+        all_options_numeric_auto['numeric'] = 'auto';
```

Comment:
You can combine these JS object/dictionary initializations even further via the spread operator. They can each be just one line:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#description